src/gui/mainwindow.ui

```diff
@@ -300,6 +301,14 @@
     <string>Display Top Toolbar</string>
    </property>
   </action>
+  <action name="actionShowStatusbar">
+   <property name="checkable">
+    <bool>true</bool>
+   </property>
+   <property name="text">
+    <string>Status &amp;Bar</string>
```

Comment:
This is from other commit (not mine) but it still conforms the other menu items (e.g. "Speed in Title Bar", "Search Engine" etc.).